docs/guide/yaml/test/example_yaml/simple-tomcat-app-sensor-test.yaml

```diff
@@ -0,0 +1,24 @@
+brooklyn.catalog:
+  id: simple-tomcat-app-sensor-test
+  version: 1.0
+  itemType: template
+  iconUrl: http://tomcat.apache.org/images/tomcat.png
+  name: Simple Tomcat Application Sensor Test
+  license: Apache-2.0
+  item:
+    brooklyn.config:
+      simple.confg: someValue
```

Comment:
Can we delete this (from here and from all the other examples)? I'm a fan of keeping the examples as simple as possible (so people don't get a false impression of how long it has to be). If we want to demonstrate other thing such as config, we can give a separate example for that.